git_hooks/pre-commit

```diff
@@ -12,17 +12,21 @@ fi
 # Jump to git root dir
 pushd ${GIT_HOME}
 
+
+localEslint=$(find -path "*/eslint-config-i2w_test-airbnb-base/node_modules/.bin/eslint" -print -quit)
 # Is eslint installed?
-which eslint > /dev/null
+which $localEslint > /dev/null
 if [ $? -ne 0 ];
 then
   echo -e "\033[41;37mPlease install ESLint:\033[0m"
-  echo "yarn global add eslint eslint-config-airbnb-base eslint-plugin-import"
+  echo "yarn add eslint@^4.19.1 https://github.com/chenwei850825/js-linter-test/ eslint-plugin-import@^2.14.0"
```

Comment:
Need to refine this yarn statement
1. Does we still need to install eslint, eslint-plugin-import by ourselves
2. "https://github.com/chenwei850825/js-linter-test/" seems to be test url